components/Search.tsx

```diff
@@ -33,18 +64,19 @@ export default function Search(props: {
         props.onClose();
     };
 
-    const {notes, error} = useSearchResultQuery("");
+    const [searchTerm, setSearchTerm] = useState("");
+    const searchResult: SearchQueryResult = useSearchResultQuery(searchTerm);
```

Comment:
I'd prefer to keep the [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment?retiredLocale=de) syntax instead of the `searchResult` variable.

Reply:
done